web/src/app/webview/src/osk/pendingLongpress.ts

```diff
@@ -1,52 +0,0 @@
-
-import {
-  type KeyElement,
-  PendingGesture,
-  VisualKeyboard
-} from 'keyman/engine/osk';
-
-import { SubkeyDelegator } from "./subkeyDelegator.js";
-
-/**
- * As control over the subkey display timer and the subkey popup are
- * both handled by the host app within the Android app, this class
- * serves mostly to communicate longpress state management from the
- * app to the VisualKeyboard.
- *
- * The `resolve()` function should be triggered by the host app
- * whenever it has recognized a completed longpress and has thus
- * begun displaying its subkey popup.  (Should these two events
- * ever become separated in time, the latter is the more critical
- * aspect.)
- */
-export class PendingLongpress implements PendingGesture {
-  private resolver: (delegator: SubkeyDelegator) => void;
-  private readonly vkbd: VisualKeyboard;
-
-  public readonly baseKey: KeyElement;
-  public readonly promise: Promise<SubkeyDelegator>;
-
-  constructor(vkbd: VisualKeyboard, e: KeyElement) {
-    this.vkbd = vkbd;
-    let _this = this;
-
-    this.promise = new Promise<SubkeyDelegator>(function(resolve) {
-      _this.resolver = resolve;
-    });
-    this.baseKey = e;
-  }
-
-  public resolve() {
-    if(this.resolver) {
-      this.resolver(new SubkeyDelegator(this.vkbd, this.baseKey));
```

Comment:
`SubkeyDelegator` may be removed as well; it's used exclusively as a follow-up to this class.

Reply:
ah, I still had to keep `SubkeyDelegator` for
https://github.com/keymanapp/keyman/blob/9fbe5f6b410df501070d95b31143ffc253fa70f7/web/src/app/webview/src/keymanEngine.ts#L109

Reply:
Right, which is within the `executePopupKey` method.  Which is no longer being called, correct?

Reply:
I thought both iOS and Android will still need KeymanWeb to handle `executePopupKey`.

Reply:
gotcha. I was able to remove popVisible (it had the same gesture)

Reply:
> I thought both iOS and Android will still need KeymanWeb to handle `executePopupKey`.

Ah, I get that.  Yeah, we did leave the Swift connections to it in place, since back when we moved iOS to rely on the WebView, we were hoping Apple would fix the motivating bug.  Didn't happen, though, and we never revisited to clean out the old code.  Probably worth making into a cleanup issue.